README.rst

```diff
@@ -1,3 +1,4 @@
+This project is tested with BrowserStack.
```

Comment:
Why?
Also, no, it's not. That tool is in no way a part of the CI stack for this project.